quill-cassandra-lagom/src/main/scala/io/getquill/CassandraLagomAsyncContext.scala

```diff
@@ -0,0 +1,64 @@
+package io.getquill
+
+import akka.Done
+import com.lightbend.lagom.scaladsl.persistence.cassandra.CassandraSession
+import io.getquill.context.cassandra.CassandraLagomSessionContext
+import io.getquill.monad.ScalaFutureIOMonad
+import io.getquill.util.ContextLogger
+
+import scala.concurrent.{ ExecutionContext, Future }
+
+class CassandraLagomAsyncContext[N <: NamingStrategy](
```

Comment:
There's too much copy-and-paste here. Please create a generic implementation that can be shared between datastax and lagom and only specializes the driver implementation. You could create an interface that needs to be implemented by the driver modules